### ISSUE TITLE
[Feature Request] Option to disable video background in the GUI

### ISSUE THREAD

**Issue by yaymuffins** [UltraStar-Deluxe/USDX#12]
Hello

It would be nice to have an option to disable video background on the songs, some low end computer might have issue playing them, while the game would be perfectly playable without them.

YayPonies/YayMuffins

**Comment by RattleSN4K3** [comment]
Hi.

Have you checked the config file? There is an option to disable video.
```
[Graphics]
VideoEnabled=Off
```

Does it fulfill your need? Please, report back.

I personally agree that having an option in the game itself is a benefit. A slightly related pull request: #85

**Comment by basisbit** [comment]
@RattleSN4K3 yes, this is part of what we need. @yaymuffins and I talked already about it in the irc channel. In games options graphics there should be an additional option for enabling / disabling video playback. this should switch both, VideoEnabled and VideoPreview, to On or Off. this also requires changes to the theme files and thus we should at least do the theme files changes before the 1.3beta release next week.

**Comment by RattleSN4K3** [comment]
> Doesn't really have anything to do with #85.

It does (as said slightly but it's there). Currently...
- If you have video disabled and visualization off, you will either see the background image or blank screen.
- If you have video disabled and visualization on, you will always see visualization.
- If you have video disabled and visualization only if no video, **you will always see visualization**.

The markup is changed with #85. In case you like to play without videos, you likely want to see the set background image **or** a visualization, depending on if an image file is available. In case of fully compat to low end computers, there is still the option `Visualization=Off`, `Visualization=WhenNoVideo` is simply not relevant with `VideoEnabled=Off`.

**Comment by RattleSN4K3** [comment]
> this also requires changes to the theme files and thus we should at least do the theme files changes before the 1.3beta release next week.

Yep. There are already some theme changes.

**Comment by basisbit** [comment]
> There are already some theme changes.

what do you mean by that? don't see any changes for that option anywhere in the themes files

**Comment by RattleSN4K3** [comment]
"some". Not for that option. I meant, there are other changes to the Modern/Deluxe basic layout. By using other themes, these have to be adapted. _MusicGain_ for instance.

**Comment by RattleSN4K3** [comment]
Nothing against it. I've just said, that a change to the themes is essential and even required at this stage.

**Comment by natrius** [comment]
Is that already happened? Otherwise, where can i get a current, good, theme?

**Comment by dgruss** [comment]
close as not planned?

**Comment by barbeque-squared** [comment]
We should have these options be _way_ more accessible than being buried in some config file. But we should also rethink what options would actually make sense, because I think currently there is at least:
* VideoEnabled (probably just disables video entirely?)
* VideoPreview (I alsways have this one off as it makes navigation in the area of a billion times faster, I don't know why this isn't the default)
* there might also be one for disabling videos coming from themes? I don't actually know
* Then there is this whole visualization mess on top of it (does VideoEnabled=false mean it's going to now do visualizations for everything? what is actually the default for this setting?)

I think there's also still remnants of some half-video size somewhere in one of the settings, I don't know if that was done for performance or visual reasons but I've never seen anyone use it. Some of the options appear to depend on other options (which we can't really model in our UI) but some of the individual options also just don't make any sense.

But video isn't the only thing these low-end devices struggle with anyway. The first thing I do in anyone's USDX is also disable the screen fading animation thing because that isn't particularly resource-friendly either. It's especially bad combined with preview videos.

This issue should definitely remain open until there's a saner way than editing ini's to make the game more playable on low-end devices, but there should be some consideration of what options and defaults we actually _need_ before blindly dumping "the everything" into options screens.
